src/cls/rbd/cls_rbd.cc

```diff
@@ -1455,6 +1455,34 @@ int get_snapshot_name(cls_method_context_t hctx, bufferlist *in, bufferlist *out
   return 0;
 }
 
+int get_snapshot_time_stamp(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
```

Comment:
Trying to get all new functions in the "<noun>_<verb>" convention so that similar functions are grouped together. There are existing "snapshot_XYZ" functions that this should be grouped with.